packages/perseus/src/components/math-input.jsx

```diff
@@ -1,174 +1,50 @@
-/* eslint-disable react/sort-comp */
```

Comment:
Most of the changes in this file are because I reordered the component to conform to this linting rule.